web/client/components/map/openlayers/DrawSupport.jsx

```diff
@@ -809,7 +809,7 @@ export default class DrawSupport extends React.Component {
                     newDrawMethod = "Polygon";
                     const radius = previousFt && previousFt.getProperties() && previousFt.getProperties().radius || 10000;
                     let center = event.coordinate;
-                    const coords = this.polygonCoordsFromCircle(center, 100);
+                    const coords = this.polygonCoordsFromCircle(center, radius);
```

Comment:
hi. @kasongoyo can you add a test to prevent the previous problem?
maybe adding a new case where radius is different from 100

Reply:
Ok. @MV88

Reply:
@MV88 I find difficult to externalize the behaviour to make it testable. As you can see the best way is to introduce `previousFt` and spy the `polygonCoordsFromCircle` fn in test case.  I feel like the efforts required does not worth the value produced. Let me know your opinion on this